lib/generators/module_component/templates/app/services/configuration.rb.erb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module <%= class_name %>
+  class Configuration < Common::Client::Configuration::REST
+    def base_path
+      Settings.<%= class_name.downcase %>.url
+    end
+
+    def service_name
+      '<%= class_name.upcase %>/HelloWorld'
+    end
+
+    def connection
+      @conn ||= Faraday.new(base_path, headers: base_request_headers, request: request_options) do |faraday|
```

Comment:
Should the memoization variable be the same as the method name?

Reply:
✔️